From 592c915df252932961a4151c891da58c48a8db90 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 2 Mar 2023 18:47:40 -0800
Subject: [PATCH] Define CARD32 as uint as it is 32-bit

long can be 64bit on LP64 systems

Fixes build with clang-16
thunk.c:147:3: error: incompatible function pointer types initializing 'x86emuu32 (*)(X86EMU_pioAddr)' (aka 'unsigned int (*)(unsigned short)') with an expression of type 'unsigned long (*)(unsigned short)' [-Wincompatible-function-pointer-types]
                (&x_inl),
                ^~~~~~~~

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 x86emu/include/xf86int10.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/x86emu/include/xf86int10.h b/x86emu/include/xf86int10.h
index 9c131dd..acd5114 100644
--- a/x86emu/include/xf86int10.h
+++ b/x86emu/include/xf86int10.h
@@ -18,7 +18,7 @@
 
 #define CARD8  unsigned char
 #define CARD16 unsigned short
-#define CARD32 unsigned long
+#define CARD32 unsigned int
 #define pointer void *
 #define IOADDRESS void *
 #define Bool int
-- 
2.39.2

